protocol/x/prices/keeper/slinky_adapter_test.go

```diff
@@ -113,3 +113,24 @@ func TestGetPrevBlockCPCounter(t *testing.T) {
 	require.NoError(t, err)
 	require.Equal(t, uint64(marketNumber), cpCounter)
 }
+
+func TestGetAllCurrencyPairs(t *testing.T) {
```

Comment:
nit: Perhaps also a test that this returns less curreny paris if one has an invalid format?